aksel.nav.no/website/pages/templates/soknad-introside/intro-med-alternativ-soknad.tsx

```diff
@@ -0,0 +1,378 @@
+import { useEffect } from "react";
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Accordion,
+  Bleed,
+  BodyLong,
+  BodyShort,
+  Box,
+  Button,
+  Checkbox,
+  GuidePanel,
+  Heading,
+  Link,
+  List,
+  Page,
+  Show,
+  Stack,
+  VStack,
+} from "@navikt/ds-react";
+
+function Example() {
+  useDekorator();
+
+  return (
+    <Page data-aksel-template="form-intropage-v1" footer={<Footer />}>
+      <Header />
+      <VStack as="main" gap="8">
+        <Page.Block width="text" gutters>
+          <Bleed marginInline={{ lg: "24" }}>
+            <Stack
+              align="center"
+              gap="6"
+              direction={{ xs: "row-reverse", lg: "row" }}
+              justify="start"
+              wrap={false}
+            >
+              <Show above="sm">
+                <ApplicationPictogram title="Application" />
+              </Show>
+              <VStack gap="1">
+                <BodyShort size="small">NAV 10-07.03 (om relevant)</BodyShort>
+                <Heading level="1" size="large">
+                  Søknad om [ytelse]
+                </Heading>
+              </VStack>
+            </Stack>
+          </Bleed>
+        </Page.Block>
+        <Page.Block width="text" gutters>
+          <VStack gap="12">
+            <GuidePanel poster>
+              <Heading level="2" size="medium" spacing>
+                Hei, [Navn Navnesen]!
+              </Heading>
+              <BodyLong spacing>
+                Seksjonen GuidePanel brukes til en kort, overordnet veiledning
+                til søkeren. Seksjonen henter inn søkerens navn, og gir en en
+                komprimert forklaring av pengestøtten, tiltaket eller
+                hjelpemiddelet. Denne teksten hentes fra ingressen til
+                produktsiden på nav.no.
+              </BodyLong>
+              <BodyLong>
+                Avslutt teksten i seksjonen med en lenke til produktsiden på
+                nav.no som åpnes i en ny fane.
+              </BodyLong>
+            </GuidePanel>
+            <div>
+              <Heading level="2" size="large" spacing>
+                Her kan du søke om
+              </Heading>
+              <BodyLong spacing>
+                Dette feltet brukes hvis det kan være utfordrende å finne riktig
+                søknad, for eksempel fordi det er mange søknader som er nesten
+                lik hverandre. Bruk feltet til en kort og tydelig forklaring på
+                hva denne søknaden gjelder.
+              </BodyLong>
+              <Heading level="3" size="medium" spacing>
+                Dette kan du ikke søke om her
+              </Heading>
+              <BodyLong spacing>
+                Legg inn informasjon om hva du ikke kan søke om fra denne siden
+                og legg inn lenke til de aktuelle søknadene man skal bruke i
+                stedet.
+              </BodyLong>
```

Comment:
I would not use `spacing` here, since spacing is also added by the VStack.
```suggestion
              <BodyLong>
                Legg inn informasjon om hva du ikke kan søke om fra denne siden
                og legg inn lenke til de aktuelle søknadene man skal bruke i
                stedet.
              </BodyLong>
```